Portfolio/index.css

```diff
@@ -1 +1,64 @@
-/* Write your code below for PORTFOLIO */
+/* Write your code below for PORTFOLIO */
+*{
+      color: #515050;
+          margin: 0%;
+    padding: 0%;
+}
+.cont1{
+    background-color: #90A4AE;
+    margin: 0%;
+    padding: 10px;
+  
+}
+nav ul{
+    padding: 0px;

```

Comment:
Consider using 'padding: 0;' and 'margin: 0;' instead of 'padding: 0px;' and 'margin: 0px;' for consistency